libraries/core-react/src/components/Typography/Typography.tsx

```diff
@@ -90,6 +91,14 @@ const StyledTypography = styled.p<StyledProps>`
         text-overflow: ellipsis;
       }
     `}
+  ${({ link }) =>
+    link &&
```

Comment:
How does the css look when link is `false` or `undefined`?

Reply:
As expected